src/app/hooks/useMASToken.ts

```diff
@@ -1,18 +0,0 @@
-import {} from "react";
-import { useConfig } from "@rhoas/app-services-ui-shared";
-import { buildTokenEndPointUrl } from "@app/utils";
-
-export const useMASToken = () => {
```

Comment:
Would that affect other repositories or it is just used in app-services-ui

Reply:
It will affect kas-ui and srs-ui as we are passing token_url as props.